src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/LeftAlignAndTrimVariants.java

```diff
@@ -0,0 +1,470 @@
+ * leaving them with a minimum representation. The same indel can often be placed at multiple positions and still
+ * represent the same haplotype. While the standard convention with VCF is to place an indel at the left-most position
+ * this isn't always done, so this tool can be used to left-align them. This tool optionally splits multiallelic
+ * sites into biallelics and left-aligns individual alleles. Optionally, the tool will not trim common bases from indels.
+ * </p>
+ *
+ * <h3>Input</h3>
+ * <p>
+ * A variant call set to left-align and trim.
+ * </p>
+ *
+ * <h3>Output</h3>
+ * <p>
+ * A left-aligned VCF.
+ * </p>
+ *
+ * <h3>Usage examples</h3>
+ *
+ * <h4>Left align and trim alleles</h4>
+ * <pre>
+ * gatk LeftAlignAndTrimVariants \
+ *   -R reference.fasta \
+ *   -V input.vcf \
+ *   -O output.vcf
+ * </pre>
+ *
+ * <h4>Left align and don't trim alleles</h4>
+ * <pre>
+ * gatk LeftAlignAndTrimVariants \
+ *   -R reference.fasta \
+ *   -V input.vcf \
+ *   -O output.vcf \
+ *   --dont-trim-alleles
+ * </pre>
+ *
+ * <h4>Left align and trim alleles, process alleles <= 208 bases</h4>
+ * <pre>
+ * gatk LeftAlignAndTrimVariants \
+ *   -R reference.fasta \
+ *   -V input.vcf \
+ *   -O output.vcf \
+ *   --max-indel-length 208
+ * </pre>
+ * <h4>Split multiallics into biallelics, left align and trim alleles</h4>
+ * <pre>
+ * gatk LeftAlignAndTrimVariants \
+ *   -R reference.fasta \
+ *   -V input.vcf \
+ *   -O output.vcf \
+ *   --split-multi-allelics
+ * </pre>
+ *
+ * <h4>Split multiallelics into biallics, left align but don't trim alleles, and store the original AC, AF, and AN values</h4>
+ * <pre>
+ * gatk LeftAlignAndTrimVariants \
+ *   -R reference.fasta \
+ *   -V input.vcf \
+ *   -O output.vcf \
+ *   --split-multi-allelics \
+ *   --dont-trim-alleles
+ *   --keep-original-ac
+ * </pre>
+ * <h4> Left align variants up to 2000 bases to the left (default is at most left aligning 1000 bases to left)</h4>
+ * <pre>
+ * gatk LeftAlignAndTrimVariants \
+ *   -R reference.fasta \
+ *   -V input.vcf \
+ *   -O output.vcf \
+ *   --max-leading-bases 2000
+ * </pre>
+ */
+@CommandLineProgramProperties(
+        summary = "This tool takes a VCF file, left-aligns the indels and trims common bases from indels," +
+                "leaving them with a minimum representation. The same indel can often be placed at multiple positions and still" +
+                "represent the same haplotype. While the standard convention with VCF is to place an indel at the left-most position" +
+                "this isn't always done, so this tool can be used to left-align them. This tool optionally splits multiallelic" +
+                "sites into biallelics and left-aligns individual alleles. Optionally, the tool will not trim common bases from indels.",
+        oneLineSummary = "Left align and trim vairants",
+        programGroup = VariantManipulationProgramGroup.class
+)
+@DocumentedFeature
```

Comment:
Depending on the ultimate use case, you might consider marking the tool as beta for now as it is a new port of this tool that might be prone to further changes/feature requests.